### DISCUSSION TITLE
Trimble data format is not supported

### DISCUSSION THREAD

**Discussion by gvaruna** [tomojitakasu/RTKLIB#699] [Q&A]
How to use Data from Trimble Receivers directly in the program as it was not listed in the format types?

**Comment by reid-p** [comment]
I use Trimble's software to convert their data to rinex and then use that.

**Reply by gvaruna** [reply]
Thanks a lot for sharing the idea. But that's possible for post processing. Actually, I was looking for RTK solution. Thank you once again

**Comment by reid-p** [comment]
That I haven't done. I've used trimble gps as a base (with them transmitting RTCM to do rtk with other receivers), but haven't used them as a rover with rtklib doing the processing for rtk.
